lib/modules/apostrophe-seo-doc-type-manager/index.js

```diff
@@ -7,15 +7,15 @@ module.exports = {
           name: 'seoTitle',
           label: 'Title',
           type: 'string',
-          max: 60,
+          seoSoftMax: 60,
```

Comment:
If we're backing off the reduction, we can still keep the old `max`, right? Here and on `seoDescription` (`min` there, too).